server/webapp/WEB-INF/rails/webpack/views/pages/users.tsx

```diff
@@ -240,18 +273,42 @@ export class UsersPage extends Page<null, State> {
              });
   }
 
-  private bulkUpdateSystemAdmins(vnode: m.Vnode<null, State>, json: BulkUpdateSystemAdminJSON): void {
+  private updateSystemAdminPrivilegeForUser(vnode: m.Vnode<null, State>,
+                                            json: BulkUpdateSystemAdminJSON,
+                                            user: User): void {
     AdminsCRUD.bulkUpdate(json)
               .then((apiResult) => {
                 apiResult.do((successResponse) => {
+                  vnode.state.userViewStates[user.loginName()] = {
+                    updateOperationStatus: UpdateOperationStatus.SUCCESS
+                  };
+
+                  setTimeout(() => {
```

Comment:
can we remember the last timeout, and clear it if required before creating the new timeout?

This is required if the user performs multiple updates within `CLEAR_USER_UPDATE_STATUS * 1000` interval.